Routing/Routing.csproj

```diff
@@ -1,7 +1,12 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <NullableContextOptions>enable</NullableContextOptions>
```

Comment:
Remove this as it is no longer needed.

Reply:
https://github.com/messerli-informatik-ag/routing/pull/6/commits/370cb929460c1039c268cf4b22e4776d4678ac27